.github/workflows/continuous-integration.yml

```diff
@@ -6,15 +6,18 @@
 name: CI (python)
 on: [push, pull_request]
 
+env:
+  COVERAGE_OS: ubuntu-latst
```

Comment:
Typo - should this be `ubuntu-latest`?

Reply:
Yes. It was a typo after I had it working!